frontend/src/pages/org/crawl-detail.ts

```diff
@@ -704,7 +716,11 @@ ${this.crawl?.notes}
                   <li
                     class="flex justify-between p-3 border-t first:border-t-0"
                   >
-                    <div class="whitespace-nowrap truncate">
+                    <div class="whitespace-nowrap truncate flex items-center">
+                      <sl-icon
+                      name="file-earmark-zip-fill"
```

Comment:
Can you check if the formatter is running via githooks or in your editor? Looks like it's not indenting properly